MonoGame.Framework/Utilities/ReflectionHelpers.cs

```diff
@@ -2,60 +2,61 @@
 using System;
 using System.IO;
 using System.Reflection;
+using System.Runtime.InteropServices;
 
 namespace Microsoft.Xna.Framework.Utilities
 {
-	internal static class ReflectionHelpers
-	{
-		public static bool IsValueType(Type targetType)
-		{
-			if (targetType == null)
-			{
-				throw new NullReferenceException("Must supply the targetType parameter");
-			}
+    internal static class ReflectionHelpers
+    {
+        public static bool IsValueType(Type targetType)
+        {
+            if (targetType == null)
+            {
+                throw new NullReferenceException("Must supply the targetType parameter");
+            }
 #if WINRT
-			return targetType.GetTypeInfo().IsValueType;
+            return targetType.GetTypeInfo().IsValueType;
 #else
 			return targetType.IsValueType;
 #endif
-		}
-
-		public static Type GetBaseType(Type targetType)
-		{

```

Comment:
Updated.
Although long term this still leaves a risk for when the function is deprecated in modern .NET
IndexBuffer and VertexBuffer should really be updated with Generics, however this would be a breaking change.